### DISCUSSION TITLE
use oci manifests in the future

### DISCUSSION THREAD

**Discussion by Zoey2936** [nextcloud/all-in-one#2903] [Ideas] [answered]
https://github.com/nextcloud-releases/all-in-one/blob/7cff84a6f95365a3295d0e7e987c69b30612729a/.github/workflows/build_images.yml#L75
maybe then https://docs.docker.com/engine/reference/commandline/buildx_imagetools_create could be used inside the push beta workflow

**Comment by szaimen** [comment]
What is the advantage of using oci manifests?

**Reply by Zoey2936** [reply]
none, but the old may deprecate in the future

**Reply by szaimen** [reply]
I see. Then lets keep this open for the meantime so that we dont forget it.

**Reply by szaimen** [reply]
a PR would be welcome btw :)

**Reply by Zoey2936** [reply]
so you want to switch already now?

**Reply by szaimen** [reply]
I mean we could wait… Do you think the format is not stable enough yet?

**Reply by Zoey2936** [reply]
not sure if we support them long enough https://github.com/nextcloud/all-in-one/blob/5bbec81ff88754bd67596d75f7aee1b69cd496c9/php/src/Docker/DockerHubManager.php#L42C1-L42C63

**Reply by szaimen** [reply]
True, was added with https://github.com/nextcloud/all-in-one/commit/2469461a3cc07c5a6a6e965c9c3c17a018d6fd9b. So probably we should still wait a few months? WDYT?

**Reply by Zoey2936** [reply]
yes

**Reply by szaimen** [reply]
all right 👍

**Comment by szaimen** [comment, marked as answer]
I've created an issue for tracking: https://github.com/nextcloud/all-in-one/issues/3182
